### PR TITLE
VS-2805 create in focus screen

### PR DESCRIPTION
**Pull request by vetrabmercedes**
https://cabidev.atlassian.net/browse/VS-2805

### CODE REVIEW

File: react/features/filmstrip/components/native/InFocusView.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+import styles from './styles';
+
+type Props = {
+    mainUser: Object,
+    sideUser: Object,
+    isSideUserAudioMuted: boolean
+}
+
+const NAME_PLACEHOLDER = 'FALL \'20 COLLECTION';
+const UNKNOWN_NAME = 'UNKNOWN';
```
Comment (kecskesk) [inline react/features/filmstrip/components/native/InFocusView.js]: NAME_PLACEHOLDER should come from tap
UNKNOWN NAME should be empty string

**Review by kecskesk — APPROVED** [review]

**Review by zongolista — APPROVED** [review]

File: react/features/filmstrip/components/native/InFocusView.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+}
+
+function _createDefaultMainUserName() {
+    return (<View
+        style = { styles.buttonPlaceholder } >
+        <Text style = { styles.nameComponent }>{NAME_PLACEHOLDER}</Text>
+    </View>);
+}
+
+function _createShowButtonsPlaceholder() {
```
Comment (molnartimi) [inline react/features/filmstrip/components/native/InFocusView.js]: Please rename it to `_createWrapUpButtonsPlaceholder`

File: react/features/filmstrip/components/native/InFocusView.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    return (<View
+        style = { styles.buttonPlaceholder } />);
+}
+
+function _createBottomVideoComponent(sideUser, isSideUserAudioMuted) {
```
Comment (molnartimi) [inline react/features/filmstrip/components/native/InFocusView.js]: Please rename it to `_createSelfFrameVideoComponent`

File: react/features/filmstrip/components/native/InFocusView.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+            && <View style = { styles.microphoneViewStyle } >
+                <Image
+                    source = { require('../../../../../resources/img/muted_microphone.png') }
+                    style = { styles.microphoneIconStyle } />
+            </View>}
+        </View>
+    </View>);
+}
+
+function _createMainUserComponent(mainUser) {
```
Comment (molnartimi) [inline react/features/filmstrip/components/native/InFocusView.js]: Please rename it to `_createInFocusVideoComponent`

File: react/features/filmstrip/components/native/InFocusView.js [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+    return (<Thumbnail
+        isAvatarCircled = { true }
+        participant = { mainUser }
+        renderDisplayName = { true }
+        styleOverrides = { styles.inFrontBackView }
+        tileView = { true } />);
+}
+
+function _createDefaultMainUserComponent() {
```
Comment (molnartimi) [inline react/features/filmstrip/components/native/InFocusView.js]: Please rename it to `_createTemplateImageComponent`

File: react/features/filmstrip/components/native/InFocusView.js [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+/**
+ * Function that maps parts of Redux state tree into component props.
+ *
+ * @param {Object} state - Redux state.
+ * @param {Props} ownProps - Properties of component.
+ * @returns {Object}
+ */
+function _mapStateToProps(state, ownProps) {
+    const { mainUser, sideUser } = ownProps;
```
Comment (molnartimi) [inline react/features/filmstrip/components/native/InFocusView.js]: Please use `inFocusUser` and `localUser` naming instead in whole file

File: react/features/filmstrip/components/native/InFocusView.js [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+        participant = { mainUser }
+        renderDisplayName = { true }
+        styleOverrides = { styles.inFrontBackView }
+        tileView = { true } />);
+}
+
+function _createDefaultMainUserComponent() {
+    return (<Image
+        source = {{
+            uri: 'https://media.cliotest.com/VS/0da11b19-985d-497b-a532-c6120f4dec5f.png'
```
Comment (molnartimi) [inline react/features/filmstrip/components/native/InFocusView.js]: Please add a TODO comment to make this URL settable by tabletparty.

File: react/features/filmstrip/components/native/TileView.js [hunk 8]
```diff
@@ -103,7 +103,15 @@ class TileView extends Component<Props> {
     render() {
         const { _height, _width, onClick } = this.props;
         const rowElements = this._groupIntoRows(this._renderThumbnails(), COLUMN_COUNT);
-        const pageOrderedThumbnails = this._groupThumbnailsByPages(rowElements);
+        const mainUser = this.props._participants[1];
```
Comment (molnartimi) [inline react/features/filmstrip/components/native/TileView.js]: Please use `inFocusUser` and `localUser` naming here, too.
Also add a TODO comment to make inFocusUser settable by tabletparty later.
We need to put always the local participant to the self frame, is she always the first in the participant list?

Comment (vetrabmercedes) [reply react/features/filmstrip/components/native/TileView.js]: Local participant is always the first one, but when we implement the shorting ticket, we will modify the selection logic.

File: react/features/filmstrip/components/native/styles.js [hunk 9]
```diff
@@ -141,6 +141,81 @@ export default {
...
+    },
+
+    inFrontBackView: {
+        position: 'absolute',
+        justifyContent: 'flex-end',
+        width: '100%',
+        height: '100%'
+    },
+
+    gradientOverlay: { position: 'absolute',
```
Comment (molnartimi) [inline react/features/filmstrip/components/native/styles.js]: missing enter after {

**Review by zongolista — APPROVED** [review]

**Pull request merged**